github/search_test.go

```diff
@@ -914,3 +914,28 @@ func TestLabelsSearchResult_Marshal(t *testing.T) {
 
 	testJSONMarshal(t, u, want)
 }
+func TestCommitResult_Marshal(t *testing.T) {
```

Comment:
Please add a blank line before this test function.

Reply:
added a blank line, Thanks for suggesting changes.